packages/picasso.js/src/core/chart-components/container/container.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @typedef {object} ContainerComponent
```

Comment:
Rename to `ComponentContainer` to be consistent with other components

Reply:
done